gcore/resource_gcore_cdn_options.go

```diff
@@ -496,6 +496,56 @@ var (
 				},
 			},
 		},
+		"proxy_connect_timeout": {
+			Type:        schema.TypeList,
+			MaxItems:    1,
+			Optional:    true,
+			Computed:    true,
+			Description: "The time limit for establishing a connection with the origin.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"enabled": {
+						Type:     schema.TypeBool,
+						Optional: true,
+						Default:  true,
+					},
+					"values": {
+						Type:     schema.TypeMap,
+						Optional: true,
+						Computed: true,
+						DefaultFunc: func() (interface{}, error) {
+							return map[string]interface{}{}, nil
+						},
+						Elem: schema.TypeString,
+					},
+				},
+			},
+		},
+		"proxy_read_timeout": {
+			Type:        schema.TypeList,
+			MaxItems:    1,
+			Optional:    true,
+			Computed:    true,
+			Description: "The time limit for receiving a partial response from the origin. If no response is received within this time, the connection will be closed.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"enabled": {
+						Type:     schema.TypeBool,
+						Optional: true,
+						Default:  true,
+					},
+					"values": {
```

Comment:
`values`? As I know it should be a single string value?

Reply:
the same for `proxy_connect_timeout`. bad copy paste

Reply:
Thanks, I got confused with another option